docs.openc3.com/docs/getting-started/cli.md

```diff
@@ -0,0 +1,212 @@
+2025/03/22 19:50:40.429 (SCRIPTRUNNER): Script config/DEFAULT/targets/INST/procedures/stash.rb spawned in 0.796683293 seconds <ruby 3.2.6>
+2025/03/22 19:50:40.453 (SCRIPTRUNNER): Starting script: stash.rb, line_delay = 0.1
+At [INST/procedures/stash.rb:3] state [running]
+At [INST/procedures/stash.rb:4] state [running]
+2025/03/22 19:50:40.732 (stash.rb:4): key1: val1
+At [INST/procedures/stash.rb:5] state [running]
+At [INST/procedures/stash.rb:6] state [running]
+2025/03/22 19:50:40.936 (stash.rb:6): key2: val2
+At [INST/procedures/stash.rb:7] state [running]
+2025/03/22 19:50:41.039 (stash.rb:7): CHECK: 'val1' == 'val1' is TRUE
+At [INST/procedures/stash.rb:8] state [running]
+2025/03/22 19:50:41.146 (stash.rb:8): CHECK: 'val2' == 'val2' is TRUE
+At [INST/procedures/stash.rb:9] state [running]
+2025/03/22 19:50:41.256 (stash.rb:9): CHECK: '["key1", "key2"]' == '["key1", "key2"]' is TRUE
+At [INST/procedures/stash.rb:10] state [running]
+At [INST/procedures/stash.rb:11] state [running]
+At [INST/procedures/stash.rb:12] state [running]
+2025/03/22 19:50:41.556 (stash.rb:12): CHECK: '{"key1"=>1, "key2"=>2}' == '{"key1"=>1, "key2"=>2}' is TRUE
+At [INST/procedures/stash.rb:13] state [running]
+At [INST/procedures/stash.rb:14] state [running]
+2025/03/22 19:50:41.763 (stash.rb:14): CHECK: true == true is TRUE
+At [INST/procedures/stash.rb:15] state [running]
+At [INST/procedures/stash.rb:16] state [running]
+At [INST/procedures/stash.rb:17] state [running]
+At [INST/procedures/stash.rb:18] state [running]
+2025/03/22 19:50:42.176 (stash.rb:18): CHECK: '[1, 2, [3, 4]]' == '[1, 2, [3, 4]]' is TRUE
+At [INST/procedures/stash.rb:19] state [running]
+At [INST/procedures/stash.rb:21] state [running]
+At [INST/procedures/stash.rb:22] state [running]
+At [INST/procedures/stash.rb:23] state [running]
+2025/03/22 19:50:42.587 (stash.rb:23): CHECK: '{"one"=>1, "two"=>2, "string"=>"string"}' == '{"one"=>1, "two"=>2, "string"=>"string"}' is TRUE
+At [INST/procedures/stash.rb:24] state [running]
+2025/03/22 19:50:42.697 (SCRIPTRUNNER): Script completed: stash.rb
+At [INST/procedures/stash.rb:0] state [stopped]
+script complete
+%
+```
+
+## Validate
+
+Validate is used to validate built COSMOS plugins. It walks through the installation process without actually installing the plugin.
+
+```bash
+% openc3.sh cli validate openc3-cosmos-test-1.0.0.gem
+Installing openc3-cosmos-test-1.0.0.gem
+Successfully validated openc3-cosmos-test-1.0.0.gem
+```
+
+## Load
+
+Load can load a plugin into COSMOS without using the GUI. This is useful for scripts or CI/CD pipelines.
+
+```bash
+% openc3.sh cli load openc3-cosmos-test-1.0.0.gem
+Installing openc3-cosmos-test-1.0.0.gem
+Successfully installed openc3-cosmos-test-1.0.0.gem
+```
+
+## List
+
+List displays all the installed plugins.
+
+```bash
+% openc3.sh cli load openc3-cosmos-test-1.0.0.gem
+Installing openc3-cosmos-test-1.0.0.gem
+Successfully installed openc3-cosmos-test-1.0.0.gem
```

Comment:
This is a copy and paste error but I had issues running load and list that might need an update before merging.